sentry/src/main/java/io/sentry/DefaultSentryClientFactory.java

```diff
@@ -53,6 +53,14 @@
      * Default timeout of an HTTP connection to Sentry.
      */
     public static final int TIMEOUT_DEFAULT = (int) TimeUnit.SECONDS.toMillis(1);
```

Comment:
Please rename this to `CONNECTION_TIMEOUT_DEFAULT` to help differentiate with the new one.

Reply:
Yeah, I wasn't sure how much I should change and kept the changes as minimal as possible. Happy to change it though